src/slacko.ml

```diff
@@ -599,12 +599,16 @@ let optionally_add key value uri = match value with
 
 let definitely_add key value = optionally_add key (Some value)
 
-let unauthed_endpoint ?(base_url=default_base_url) e =
+let unauthed_endpoint ~base_url e =
```

Comment:
Having the default set there made it possible to accidentally forget to pass it in one of the functions that doesn't take a `session` and thus requires an optional `base_url` arg. I found this while running some tests through a local mitmproxy (to more easily inspect the responses slack was returning) because I'd forgotten to pass `base_url` to `unauthed_endpoint` somewhere and the request wasn't showing up in my proxy.